webpack.config.js

```diff
@@ -340,6 +341,80 @@ const settingsPage = {
 	],
 };
 
+const styles = {
+	...sharedConfig,
+	entry: {
+		'admin-tables': './assets/css/src/admin-tables.css',
+		'amp-block-editor': './assets/css/src/amp-block-editor.css',
+		'amp-customizer': './assets/css/src/amp-customizer.css',
+		'amp-customizer-legacy': './assets/css/src/amp-customizer-legacy.css',
+		'amp-default': './assets/css/src/amp-default.css',
+		'amp-icons': './assets/css/src/amp-icons.css',
+		'amp-mobile-version-switcher': './assets/css/src/amp-mobile-version-switcher.css',
+		'amp-playlist-shortcode': './assets/css/src/amp-playlist-shortcode.css',
+		'amp-post-meta-box': './assets/css/src/amp-post-meta-box.css',
+		'amp-validation-error-taxonomy': './assets/css/src/amp-validation-error-taxonomy.css',
+		'amp-validation-single-error-url': './assets/css/src/amp-validation-single-error-url.css',
+		'amp-validation-tooltips': './assets/css/src/amp-validation-tooltips.css',
+	},
+	module: {
+		...sharedConfig.module,
+		rules: sharedConfig.module.rules.map(
+			( rule ) => {
+				const cssLoader = Array.isArray( rule.use ) && rule.use.find(
+					( loader ) => loader.loader && loader.loader.includes( '/css-loader' ),
+				);
+
+				/**
+				 * Prevent "Module not found: Error: Can't resolve ..."
+				 * being thrown for `url()` CSS rules.
+				 */
+				if ( cssLoader ) {
+					cssLoader.options = {
+						...cssLoader?.options,
+						url: false,
+					};
+				}
+
+				return rule;
+			},
+		),
+	},
+	plugins: [
+		...sharedConfig.plugins.filter(
+			( plugin ) => plugin.constructor.name !== 'DependencyExtractionWebpackPlugin',
+		),
```

Comment:
This shouldn't be necessary if the `IgnoreEmitPlugin` plugin was before the `DependencyExtractionWebpackPlugin` plugin, but this works :+1:.

Reply:
I tried to avoid adding that but haven't thought about the order of the plugins and so the `.asset.php` files were always generated. I guess we could improve the implementation in the future.